src/site.webmanifest

```diff
@@ -0,0 +1,7 @@
+{
+    "icons": [{
+        "src": "icon.png",
+        "sizes": "192x192"
```

Comment:
I talked to some folks at google and they are depending on `"type": "image/png"` to be present for web manifest to work well on Chrome for Android. This is because they are not currently able to convert from one format to an icon format supported by Android. Although `"type": "image/png"` is technically not required by the w3c spec, I would recommend adding it back into the document.